src/.env.sample

```
@@ -275,3 +278,6 @@ PROJECT_SERVICE_BASE_URL=/project/
 
 # base url of survey service
 SURVEY_SERVICE_BASE_URL=/survey/
+
+# Entity management service base name
+ENTITY_MANAGEMENT_SERVICE_NAME='/entity-management/'
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)